README.md

```diff
@@ -55,14 +64,39 @@ mkdir -p WORK_DIR/EXAMPLE_OUTPUT/
 mkdir -p WORK_DIR/EXAMPLE_PROJECT/
 ```
 
-4. Make a file with a list of images to run. For example, let's say we want to process all the `.tif` images in the source folder. We'll get the list of files, and put them into a text file in the project folder. This will be used by the workflow.
+You also need to put the `parse_csv.py` script in there, into a `templates` folder:
```

Comment:
I didn't expect that this is necessary. My expectation is that any file in `templates/` in the workflow repo will be available for use as a `template` in any process in that workflow.